apps/backend/src/modules/users/user.controller.ts

```diff
@@ -1,13 +1,17 @@
-import { APIPath } from "~/libs/enums/enums.js";
+import { type MultipartFile } from "@fastify/multipart";
+
+import { APIPath, ErrorMessage } from "~/libs/enums/enums.js";
 import {
+	type APIHandlerOptions,
 	type APIHandlerResponse,
 	BaseController,
 } from "~/libs/modules/controller/controller.js";
-import { HTTPCode } from "~/libs/modules/http/http.js";
+import { HTTPCode, HTTPError } from "~/libs/modules/http/http.js";
```

Comment:
We have special error for each module. So there we should have UserError (check examples in `\shared\src\libs\exceptions`)